### PR TITLE
Add metric: variance of (log(rewards) - logprobs) on test.

### PR DESCRIPTION
**Pull request by alexhernandezgarcia**

### CODE REVIEW

File: gflownet/gflownet.py [hunk 1]
```diff
@@ -1010,6 +1018,7 @@ def test(self, **plot_kwargs):
                 self.kl,
                 self.jsd,
                 self.corr_prob_traj_rewards,
+                self.var_logrewards_logp,
                 self.nll_tt,
```
Comment (michalkoziarski) [inline gflownet/gflownet.py:1022]:
```suggestion
                corr_prob_traj_rewards,
                var_logrewards_logp,
                nll_tt,
```
The classic.

**Review by michalkoziarski — COMMENTED** [review]
Other than one issue (that was fixed in other branches already I think), LGTM.

File: gflownet/gflownet.py (hunk 1 above)
Comment (alexhernandezgarcia) [reply gflownet/gflownet.py:1022]: Thanks! The good fix would be actually different, but the good good fix will change all this anyway.

**Comment by alexhernandezgarcia** [comment]
@michalkoziarski would you approve after committing your suggestion?

**Review by michalkoziarski — APPROVED** [review]

**Pull request merged**